src/lib/Input.svelte

```diff
@@ -45,9 +41,20 @@
         innerTextContainsLetters();
 
     const handleKeyDown = (event: KeyboardEvent) => {
-        if (!canSend || event.key !== "Enter" || event.shiftKey) return;
-        send();
-        event.preventDefault();
+        let pressedEnter: boolean = event.key === "Enter";
+        let pressedShift: boolean = event.shiftKey;
+
+        if (pressedEnter) {
+            if (pressedShift) {
```

Comment:
[Non blocker] I like to often implement this single line `if`s as:

```ts
if (pressedShift) return;
if (canSend) send();
```

Let me know if you don't prefer this though! It's all a matter of preference so I'm happy for us to agree on a common practice

Reply:
Oh yeah. I had a single line before hand, however, if it's just a single line, whenever you press enter when the message box is blank, it will also create a new line. So this way is the way for us to only have `Enter + Shift` create a new line.

Reply:
Ya, I think that sounds good! good catch! With this updated check first for `pressendEnter` I think the following implementation would be equivalent:

```ts
        if (pressedEnter) {
            if (pressedShift) return;
            if (canSend) send();
            event.preventDefault();
        }
```

Reply:
Ok. I will just test that out before I merge that in.